src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -157,7 +167,11 @@ private function doImport($resource, $type = null, bool $ignoreErrors = false, $
         } catch (FileLoaderImportCircularReferenceException $e) {
             throw $e;
         } catch (\Exception $e) {
-            if (!$ignoreErrors) {
+            if (self::IGNORE_ALL !== $ignoreErrors) {
+                if ($e instanceof FileLocatorFileNotFoundException && self::IGNORE_FILE_NOT_FOUND === $ignoreErrors) {
+                    return;
```

Comment:
we should still ensure we have a FileExistenceResource added in the resource tracking in such case.

and I suggest adding a separate catch block for FileLocatorFileNotFoundException (we won't need the wrapped in LoaderLoadException anyway)

Reply:
I have added another catch block.